### DISCUSSION TITLE
Fade is too long when switching songs in song selection

### DISCUSSION THREAD

**Discussion by Tarantemo** [ppy/osu#23012] [Q&A]
When you try to switch songs with right and left buttons its very annoying because you can hear the previous song for a bit
video: https://files.catbox.moe/iw3rz8.mp4 (lazer)
video for how its on stable: https://files.catbox.moe/7vs7jk.mp4

Not really sure if its a feature or a bug so I'm curious what other people think of this. Maybe some people like it this way but for myself its not very pleasant feeling.

Would suggest maybe making a setting with which you can turn on and off the fade or just make it like its on stable.

**Comment by Gyoshi** [comment]
I like it. It's less jarring. If you go through many songs fast it's nicer to listen to one song playing while you do so.

Though I also would prefer it update slightly quicker, perhaps like half or 2/3 the current buffer period.

**Comment by GhostylGit** [comment]
As someone who mainly uses arrow keys to scroll through maps the audio not changing instantly gets really tedious and I'm forced to slow down alot unlike stable. For me it's quite an important quality of life thing so if you could look into it that'd be cool ❤️
